src/strategy.ts

```diff
@@ -47,7 +51,8 @@ export interface Strategy {
     commits: Commit[],
     latestRelease?: Release,
     draft?: boolean,
-    labels?: string[]
+    labels?: string[],
+    bumpOnlyOptions?: BumpReleaseOptions
```

Comment:
I'm a bit unclear on what these options are for. Perhaps a docstring is good enough to describe what it's use is for.

Reply:
Yes, I will add some documentation. 

I wasn't 100% sure on this, but basically these serve as an indicator that you are creating a release with a version number update, even if there were no associated conventional commits. 

So it is:
A: An indicator that we want to make a release even though there are no calculated changes.
B: The version number we want for that release.